packages/fe-fpm-writer/src/section/types.ts

```diff
@@ -27,6 +27,13 @@ export interface InternalCustomSection extends CustomSection, CustomSubSection,
     dependencies?: string;
 }
 
+export interface CustomHeaderSection extends CustomSection {
+    /**
+     * Sub title of header section.
+     */
+    subTitle?: string;
```

Comment:
what about other properties?
Like `templateEdit`, `stashed` , `requestGroupId`, `flexSettings`

More details about properties can be find here - https://ui5.sap.com/test-resources/sap/fe/core/fpmExplorer/index.html#/customElements/customHeaderFacetContent
I see following are missing:
![image](https://github.com/SAP/open-ux-tools/assets/90789422/328271f4-9ecc-4a56-9e80-9bb40605f74c)